recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -131,13 +135,25 @@ def loose_lt_semver(v1, v2):
         if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "5":
             raise ConanInvalidConfiguration("gcc < 5 is not supported")
 
+    def _cmake_new_enough(self, required_version):
```

Comment:
Should not be required with the new portable binaries

Reply:
Ditto

Reply:
removed in https://github.com/conan-io/conan-center-index/pull/16131/commits/0235e035265f430e38dededc0f3d0293accf07f8